types/core/config.d.ts

```diff
@@ -1,141 +1,1045 @@
+       * @defaultValue `'icons/svg/oak.svg'`
+       */
+      'Oak Tree': string;
+
+      /**
+       * @defaultValue `'icons/svg/obelisk.svg'`
+       */
+      Obelisk: string;
+
+      /**
+       * @defaultValue `'icons/svg/pawprint.svg'`
+       */
+      Pawprint: string;
+
+      /**
+       * @defaultValue `'icons/svg/ruins.svg'`
+       */
+      Ruins: string;
+
+      /**
+       * @defaultValue `'icons/svg/tankard.svg'`
+       */
+      Tankard: string;
+
+      /**
+       * @defaultValue `'icons/svg/temple.svg'`
+       */
+      Temple: string;
+
+      /**
+       * @defaultValue `'icons/svg/tower.svg'`
+       */
+      Tower: string;
+
+      /**
+       * @defaultValue `'icons/svg/trap.svg'`
+       */
+      Trap: string;
+
+      /**
+       * @defaultValue `'icons/svg/skull.svg'`
+       */
+      Skull: string;
+
+      /**
+       * @defaultValue `'icons/svg/statue.svg'`
+       */
+      Statue: string;
+
+      /**
+       * @defaultValue `'icons/svg/sword.svg'`
+       */
+      Sword: string;
+
+      /**
+       * @defaultValue `'icons/svg/village.svg'`
+       */
+      Village: string;
+
+      /**
+       * @defaultValue `'icons/svg/waterfall.svg'`
+       */
+      Waterfall: string;
+      /**
+       * @defaultValue `'icons/svg/windmill.svg'`
+       */
+      Windmill: string;
+    } & Partial<Record<string, string>>;
+
+    /**
+     * @defaultValue `'fas fa-book-open'`
+     */
     sidebarIcon: string;
   };
 
+  /**
+   * Configuration for the Macro entity
+   */
+  Macro: {
+    /**
+     * @defaultValue `Macro`
+     */
+    entityClass: ConstructorOf<Macro>;
+
+    /**
+     * @defaultValue `Macros`
+     */
+    collection: ConstructorOf<Macros>;
+
+    /**
+     * @defaultValue `MacroConfig`
+     */
+    sheetClass: any; // TODO: ConstructorOf<MacroConfig>
+
+    /**
+     * @defaultValue `'fas fa-terminal'`
+     */
+    sidebarIcon: string;
+  };
+
+  /**
+   * Configuration for MeasuredTemplate settings and options
+   */
+  MeasuredTemplate: {
+    types: {
+      /**
+       * @defaultValue `'Circle'`
+       */
+      circle: string;
+
+      /**
+       * @defaultValue `'Cone'`
+       */
+      cone: string;
+
+      /**
+       * @defaultValue `'Rectangle'`
+       */
+      rect: string;
+
+      /**
+       * @defaultValue `'Ray'`
+       */
+      ray: string;
+    } & Partial<Record<string, string>>;
+    defaults: {
+      /**
+       * @defaultValue `53.13`
+       */
+      angle: number;
+
+      /**
+       * @defaultValue `1`
+       */
+      width: number;
+    };
+  };
+
   /**
    * Configuration for the default Playlist entity class
    */
   Playlist: {
+    /**
+     * @defaultValue `Playlist`
+     */
     entityClass: ConstructorOf<Playlist>;
-    sheetClass: any;
+
+    /**
+     * @defaultValue `Playlists`
+     */
+    collection: ConstructorOf<Playlists>;
+
+    /**
+     * @defaultValue `PlaylistConfig`
+     */
+    sheetClass: ConstructorOf<PlaylistConfig>;
```

Comment:
This is actually null by default. And I couldn't fine a place where it was assigned.

Reply:
I made the `@defaultValue` `null` and the type `ConstructorOf<PlaylistConfig> | null`.